test/horeka_scheduler_test/horeka_config.yml

```diff
@@ -3,8 +3,11 @@
 name: "SLURM"
 partition: "accelerated"
 job-name: "horeka_test_job"
+#horeka_template: True
```

Comment:
is this intentional? I would have thought that the horeka test config would not have this line commented out.